editor.planx.uk/src/pages/FlowEditor/lib/useScrollControlsAndRememberPosition.ts

```diff
@@ -15,97 +16,96 @@ import { rootFlowPath } from "../../../routes/utils";
 const useScrollControlsAndRememberPosition = (
   scrollContainerRef: React.RefObject<HTMLDivElement>,
 ) => {
+  const currentPath = rootFlowPath(true);
+  const isLoading = useLoadingRoute();
+
   useEffect(() => {
-    let scrollListener: Subscription;
-    let dragListener: Subscription;
+    if (isLoading) return;
 
-    const storageKey = ["scrollPos", rootFlowPath(true)].join(":");
+    const storageKey = ["scrollPos", currentPath].join(":");
 
     const container = scrollContainerRef.current;
+    if (!container) return;
```

Comment:
Added as a guard clause to drop a level of nesting below, the rest of the diff below is just - 
 - Indentation 
 - Removed `any`s, added types